charts/acp-cd/README.md

```diff
@@ -0,0 +1,70 @@
+# Cloudentity Authorization CD
+
+[Cloudentity](https://cloudentity.com/) Authorization Control Plane (ACP) is a cutting edge platform for the API access control. ACP consolidates capabilities of a modern OAuth/OIDC server with an advanced authorization, consent management, and developer enablement.
+
+This chart bootstraps [acp-cd](https://docs.authorization.cloudentity.com/guides/developer/protect/istio/configuration_import/) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
```

Comment:
replace this link when docs will be ready

Reply:
Let's link to the generic declarative configuration doc when ready

Reply:
waiting for @wkotlowski